nrespipe/main.py

```diff
@@ -61,6 +61,16 @@ def stack_nres_calibrations():
                                           settings.data_reduction_root, args.nres_instrument, target=args.target)
 
 
+def run_trace_orders():
+    parser = argparse.ArgumentParser(description='Trace two fibers across all orders of a reduced 2D flat field image.')
+    parser.add_argument('--infile', dest='infile', required=True, help='Full path to reduced input file.'
+    parser.add_argument('--outfile', dest='outfile', required=True, help='Full path for ascii output file.'
+
+    args = parser.parse_args()
+
+    tasks.run_trace_orders.delay(args.infile, args.outfile)
```

Comment:
This function does not seem to exist.

Reply:
I'm not sure how tasks work in celery; I just copied the syntax from another function. Perhaps I need to rename the `trace` function (currently living in trace_orders.py) to `run_trace_orders`.

Reply:
Replaced `run_trace_orders` in line 71 with `trace` in latest commit.